src/EventGridExtension/EventGridExtension.csproj

```diff
@@ -21,10 +21,10 @@
   </PropertyGroup>
 
   <ItemGroup Condition="$(TargetFramework)=='netstandard2.0'">
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.14" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.EventGrid" Version="1.3.0" />
+    <PackageReference Include="Microsoft.Azure.EventGrid" Version="3.2.0" />
```

Comment:
Do we know what changed here? Jumping major versions typically requires some investigation to make sure there's no breaks.

Reply:
Other than my initial work being super-old, no :)

But I did test it out w/ the UTs and a sample Function App... Happy to remove the bump in favor of just XML comments if you'd rather.

Reply:
Other than my initial work being super-old, no :)

But I did test it out w/ the UTs and a sample Function App... Happy to remove the bump in favor of just XML comments if you'd rather.